### ISSUE TITLE
Does Not Support Proper Case Naming in Defining Object

### ISSUE THREAD

**Issue by jonmchan** [rivsc/Beautiful-Scaffold#16]
The following fails in beautiful-scaffold. Please note the casing of the model name.

```
rails g scaffold Comment body:string author:string post:references
```

This works though in regular scaffold. Not sure if you are looking to be completely compatible, but this threw errors while I was trying to use this originally and took me some time to figure out why.

```
rails g beautiful-scaffold Comment body:string author:string post:references
```

```
/home/jchan/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/thor-0.19.4/lib/thor/actions/file_manipulation.rb:231:in `binread': No such file or directory @ rb_sysopen - /home/jchan/tmp/testproject/app/models/Comment.rb (Errno::ENOENT)
```

**Comment by rivsc** [comment]
Thanks, fixed on next version.

**Comment by rivsc** [comment]
Fixed on 1.0.1
